src/main/java/tictactoe/Display.java

```diff
@@ -22,7 +22,7 @@ public void showGrid(Grid grid) {
 
     public void outcomeMessage(String outcome) {
         clearScreen();
-        if (outcome.equals("TIE")) {
+        if (outcome.equals("tie")) {
```

Comment:
Could be nice to use an Enum for outcomes - that way you could have the code read:

```
if (outcome.isTie()) {
 ....
}
```

Reply:
Thanks, Matt. I'll add this